cms/envs/aws.py

```diff
@@ -597,6 +597,8 @@
         INSTALLED_APPS.index('eventtracking.django.apps.EventTrackingConfig'),
         'openedx_caliper_tracking'
     )
+CALIPER_DELIVERY_ENDPOINT = ENV_TOKENS.get('CALIPER_DELIVERY_ENDPOINT')
+CALIPER_DELIVERY_AUTH_TOKEN = ENV_TOKENS.get('CALIPER_DELIVERY_AUTH_TOKEN')
 
```

Comment:
I think the best place to put `auth token` is `*.auth.json file
and we should get it like this
`CALIPER_DELIVERY_AUTH_TOKEN = AUTH_TOKENS.get('CALIPER_DELIVERY_AUTH_TOKEN')`

Reply:
Good catch.